### ISSUE TITLE
Make Bokeh service URLs relative

### ISSUE THREAD

**Issue by jakirkham** [dask/distributed#2383]
Would be great to make these URLs relative. This is helpful when using things like nbserverproxy.

https://github.com/dask/distributed/blob/2ec428ae5652f7d068baeb18223fb8a04ab8804e/distributed/bokeh/templates/worker-table.html#L23

https://github.com/dask/distributed/blob/2ec428ae5652f7d068baeb18223fb8a04ab8804e/distributed/bokeh/templates/workers.html#L5

**Comment by mrocklin** [comment]
I agree generally, but in these two cases things might be more complex

1.  In the first case, we're referring to another server, so a relative link may not work
2.  In the second case, I think that that link is already relative

**Comment by mrocklin** [comment]
Any response here @jakirkham ?  OK to close?

**Comment by jakirkham** [comment]
> In the first case, we're referring to another server, so a relative link may not work

Would it be possible to proxy this somehow?

> In the second case, I think that that link is already relative

Hmm...not sure why this link wasn't working before, but am not currently able to reproduce it.

**Comment by mrocklin** [comment]
Maybe.  I think that @yuvipanda or @ian-r-rose may have had something that
would proxy to other servers.

**Comment by yuvipanda** [comment]
Can bokeh server take a 'base_url' parameter? This is how usually things work with nbserverproxy (now jupyter-server-proxy)

**Comment by mrocklin** [comment]
Is this likely to be the same as a prefix= parameter?  If so, then yes

**Comment by mrocklin** [comment]
In the end Bokeh server is just a Tornado server, to the answer is probably
yes.

**Comment by yuvipanda** [comment]
Can bokeh server take a 'base_url' parameter? This is how usually things work with nbserverproxy (now jupyter-server-proxy)
